src/app/lib/device/chromecast.js

```diff
@@ -24,6 +24,7 @@
         play: function (streamModel) {
             var self = this;
             var subtitle = streamModel.get('subFile');
+            var title = streamModel.get('title').substring(0,50);
```

Comment:
If the issue can be resolved by changing a `ccs` file (`styl` in our case), then this PR should be closed and a new one with the correct solution should be made.